semcore/mini-chart/src/component/score/Donut.tsx

```diff
@@ -55,6 +55,7 @@ class DonutRoot extends Component<ScoreDonutProps, {}, {}, typeof DonutRoot.enha
     }
 
     const viewBox = isSemiDonut ? '0 0 24 12' : '0 0 24 24';
+    const strokeDashoffsetBase = -1 * (valueStrokeDasharray + (isSemiDonut ? offsetPoint : 0));
```

Comment:
What this line does? Why the value is multiplied by -1?

Reply:
All stroke* starts from one point. We need to rotate strokeDashOffset to the left for the value size for start base(grey) path of chart from correct place